fairmq.sh

```diff
@@ -30,9 +31,22 @@ case $ARCHITECTURE in
     [[ ! $NANOMSG_ROOT ]] && NANOMSG_ROOT=`brew --prefix nanomsg`
     [[ ! $MSGPACK_ROOT ]] && MSGPACK_ROOT=`brew --prefix msgpack`
   ;;
+  *)
+    # Check, if we want to build the ofi transport, which is for all versions v1.4.2+
+    function fairmq_build_ofi() {
+      pushd $SOURCEDIR
+      git --no-pager tag -l --merged HEAD --contains "v1.4.2" >/dev/null 2>&1
```

Comment:
As said separately, and as you say yourself, you really cannot rely on this to work at all.

Reply:
For the record I am looking for a way to detect it properly. In any case, better to use the version variable than the `git` approach.

Reply:
I am fine to make it a manual switch. How should I implement it?
* Can you point to a recipe with an example?
* Can I just look for an environment variable?
* How can I make a `require:` item depend on the presence of an environment variable?

Reply:
Switching to the comment section because it's clearer 😄